src/main/scala/com/gu/salesforce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -24,21 +22,6 @@ trait MessageHandler extends Logging {
 
   val queueName = s"salesforce-outbound-messages-${Config.stage}"
 
-  def parseMessage(requestBody: String) = {
-    val is = new ByteArrayInputStream(requestBody.getBytes)
-    val messageFactory = MessageFactory.newInstance()
-    val soapMessage = messageFactory.createMessage(null, is)
-    val body = soapMessage.getSOAPBody
-    val jc = JAXBContext.newInstance(classOf[Notifications])
-    val unmarshaller = jc.createUnmarshaller()
-    val je = unmarshaller.unmarshal(body.extractContentAsDocument(), classOf[Notifications])
-    je.getValue()
-  }
```

Comment:
This method is now moved to the new `SOAPNotificationsParser` class for easier testing.